Bottom: 4735450a5ae55a63c6c6e80b4f985273f79e30c9
Top:    0ec31f8773ad034e2411351b407698f4a54eaedd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 22:25:35 +0100

Refresh of temporary-comment-out-some-cal

---

diff --git a/src/action.c b/src/action.c
index ea9a51f..3e1340c 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4058,7 +4058,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark ();
+//	  DrawMark ();
 	  Marked.status = false;
 	}
       else
@@ -4066,16 +4066,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark ();
+//	  DrawMark ();
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark ();
+//      DrawMark ();
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark ();
+//      DrawMark ();
     }
   return 0;
 }
diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 994afb6..d84f6a1 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -361,7 +361,7 @@ ghid_port_key_press_cb (GtkWidget * drawing_area,
   state = (GdkModifierType) (kev->state);
   mk = ghid_modifier_keys_state (&state);
 
-  ghid_show_crosshair (FALSE);
+  //ghid_show_crosshair (FALSE);
 
   handled = TRUE;		/* Start off assuming we handle it */
   switch (ksym)
@@ -466,7 +466,7 @@ ghid_port_button_press_cb (GtkWidget * drawing_area,
   ghid_note_event_location (ev);
   state = (GdkModifierType) (ev->state);
   mk = ghid_modifier_keys_state (&state);
-  ghid_show_crosshair (FALSE);
+  //ghid_show_crosshair (FALSE);
   HideCrosshair ();
 
   do_mouse_action(ev->button, mk);
@@ -475,7 +475,7 @@ ghid_port_button_press_cb (GtkWidget * drawing_area,
   RestoreCrosshair ();
   ghid_window_set_name_label (PCB->Name);
   ghid_set_status_line_label ();
-  ghid_show_crosshair (TRUE);
+  //ghid_show_crosshair (TRUE);
   if (!gport->panning)
     g_idle_add (ghid_idle_cb, NULL);
   return TRUE;
@@ -846,7 +846,7 @@ ghid_port_window_leave_cb (GtkWidget * widget,
         snavi_set_led (ghidgui->snavi, FALSE);
     }
 
-  ghid_show_crosshair (FALSE);
+  //ghid_show_crosshair (FALSE);
   out->has_entered = FALSE;
 
   ghid_screen_update ();
